README.md

```diff
@@ -108,13 +108,13 @@ ShareButtonsBundle currently works with the following social share services:
 
 ## Usage
 
-Place following Twig helper in place where you want to display share buttons bar:
+Insert the following Twig helper in place where you want to display share buttons bar:
```

Comment:
+Insert the following Twig helper in the place where you want to display the share buttons bar: